Lib/statistics.py

```diff
@@ -870,9 +870,12 @@ def f_inv(y):
     return f_inv
 
 def _quartic_invcdf_estimate(p):
+    # A handrolled piecewise approximation. There is no magic here.
     sign, p = (1.0, p) if p <= 1/2 else (-1.0, 1.0 - p)
+    if p < 0.0106:
+        return ((2.0 * p) ** 0.3838 - 1.0) * sign
     x = (2.0 * p) ** 0.4258865685331 - 1.0
-    if p >= 0.004 < 0.499:
+    if p < 0.499:
```

Comment:
Do we need the check `p < 0.499`? For `p=0.5` the expression is almost zero (up to rounding errors?) and p cannot be larger than 0.5 here.

Reply:
Yes, the purpose is to have p=0.5 give exactly zero.

Reply:
I see. With this PR the behaviour around .5 looks like this:

![image](https://github.com/user-attachments/assets/5e67b0f2-c089-4d1d-86a5-34a983674e6d)

We can improve a bit and get the behavior of current main by setting the threshold a bit higher

```suggestion
    if p < 0.49999999:
```